playground/components/header.tsx

```diff
@@ -61,88 +71,109 @@ export const Header: Component<{
-        <button
-          type="button"
-          onClick={props.toggleDark}
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
-          title="Toggle dark mode"
-        >
-          <span class="sr-only">{props.dark ? 'Light' : 'Dark'} mode</span>
-
-          <Show
-            when={props.dark}
-            fallback={
-              <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                <path
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  stroke-width="2"
-                  d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
-                />
+        <div
+          class={`z-10`}
+          style={(showMenu()) ? {
+            position: "absolute",
+            top: "56px", //Set based on header size+margin*2
+            width: "fit-content"
+          } : {}}>
+          <div class={`${showMenu() ? "flex flex-col md:flex-row justify-center bg-red-500" : "hidden"} md:items-center md:space-x-2 md:flex`}>
+            <button
+              type="button"
+              onClick={props.toggleDark}
+              class="flex flex-row space-x-1 items-center px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
+              title="Toggle dark mode"
+            >
+
+              <Show
+                when={props.dark}
+                fallback={
+                  <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
+                    />
+                  </svg>
+                }
+              >
+                <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                  <path
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    stroke-width="2"
+                    d="M12 3v1m0 16v1m9-9h-1M4 12H3m15.364 6.364l-.707-.707M6.343 6.343l-.707-.707m12.728 0l-.707.707M6.343 17.657l-.707.707M16 12a4 4 0 11-8 0 4 4 0 018 0z"
+                  />
+                </svg>
+              </Show>
+              <span class="text-xs md:sr-only">{props.dark ? 'Light' : 'Dark'} mode</span>
+            </button>
+
+            <label
+              class="flex flex-row space-x-1 items-center px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100 cursor-pointer"
+              title="Import from JSON"
+            >
+              <input type="file" class="sr-only" onChange={uploadFile} accept=".json" />
+              <Icon path={upload} class="h-6" style={{margin: "0px 0px 0px 0px"}}/>
```

Comment:
This can be written as follow:
```suggestion
              <Icon path={upload} class="h-6 m-0" />
```

Reply:
This doesn't work even with !important enabled, still places weird left margin. I set it back to style={{margin: 0}}. Can't figure out how to do this with tailwind.

Reply:
This doesn't work even with !important enabled, still places weird left margin. I set it back to style={{margin: 0}}. Can't figure out how to do this with tailwind.